src/button/button.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import * as React from 'react';
+import {
+  BaseButton as StyledBaseButton,
+  StartEnhancer as StyledStartEnhancer,
+  EndEnhancer as StyledEndEnhancer,
+} from './styled-components';
+import {getOverrideObject} from '../helpers/overrides';
+import {KIND, SHAPE, SIZE} from './constants';
+
+import type {ButtonPropsT} from './types';
+
+export default function Button({
+  children,
+  disabled,
+  startEnhancer,
+  endEnhancer,
+  overrides = {},
+  size = SIZE.default,
+  kind = KIND.primary,
+  shape = SHAPE.square,
```

Comment:
You mean to explicitly do this?
```
Button.defaultProps = {
  ...
};
```

Reply:
Yes. 

It can be useful if someone wants to create their own button wrapper because they can do things like:
```
FancyButton.defaultProps = Button.defaultProps
```

Additionally, some automated documentation tools will read defaultProps when generating documentation (though maybe they can also read default param values too?)

Reply:
Sounds good, will update!